libs/books/feature/src/lib/reading-list/reading-list.component.scss

```diff
@@ -2,7 +2,7 @@
 
 :host {
   display: block;
-  font-size: 0.75rem;
+  font-size: 1rem;
```

Comment:
I think these font-size changes ruin the look of the design. It's not necessary for accessibility anyway - check that the font sizes are responsive when you increase the size of the page instead.

Reply:
changing it back to 0.75rem. confirmed that this font size is responsive when increasing the size of the page.